public/views/content/addStudent.html

```diff
@@ -33,7 +33,7 @@
       </div>
 
       <div class="form-group">
-          <button class="btn btn-default" ng-click = "addStudent()" type="submit">Submit</button>
+          <button class="btn btn-default" ng-disabled="!newStudent.archived" ng-click = "addStudent()" type="submit">Submit</button>
```

Comment:
You only have to make one of the radio-buttons required, per [this](http://stackoverflow.com/questions/8287779/html5-how-to-use-the-required-attribute-with-a-radio-input-field) stackoverflow. The reason I don't love disabling the button is that

1. we've been using the `required` attribute elsewhere, so we should stay consistent
2. if you have multiple required fields, then writing out a long `ng-disabled` statement would be difficult.